src/Security/PostVoter.php

```diff
@@ -48,6 +48,11 @@ protected function voteOnAttribute($attribute, $post, TokenInterface $token): bo
     {
         $user = $token->getUser();
 
+        // all users are allowed to view posts
+        if ($attribute === self::SHOW) {
```

Comment:
I think this check was already done in the `supports()` method, isn't it?

Reply:
No, `supports()` only checks whether the voter should be invoked, which in this case, it does.